packages/core/src/components/pill/pill.scss

```diff
@@ -22,12 +23,16 @@
 }
 
 @include chip-container(false);
+@include pill-container();
```

Comment:
Whats the reason behind the additional mixin here? There is no other files which depending on pill styles.
Now you have to jump into an additional mixin to check the styling without any benefits from moving it into a mixin